### PR TITLE
style: transpose hp heatmap to better align the plot axes

### PR DESCRIPTION
**Pull request by hkang1**
## Description

transpose the hp heatmap matrix of scatter plots to better align the x and y axes.

Screen Shot 2021-04-14 at 8 14 14 PM

## Test Plan

- [ ] check that the axes align better when viewing HP Heatmap in a matrix.
- [ ] check that the gallery modal view cycles through the heatmap in the same order as the matrix.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Comment by apizzini** [comment]
I'm noticing something odd, which might be expected (and me ignorant). When clicking on some charts to zoom, the clicked chart vs the chart in the popup some time are not matching? (in the video attached, the 1st match, the 2nd and the 3rd doesn't, ...)

https://user-images.githubusercontent.com/5413702/114828127-d2780380-9dc9-11eb-9dea-ca21327ccc9e.mov

File: webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpHeatMaps.tsx [hunk 1]
```diff
@@ -240,22 +240,22 @@ const HpHeatMaps: React.FC<Props> = ({
                   minItemWidth={resize.width > 320 ? 35 : 27}
```
Comment (apizzini) [inline webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpHeatMaps.tsx:240]: I'm noticing something odd, which might be expected (and me ignorant). When clicking on some charts to zoom, the clicked chart vs the chart in the popup some time are not matching? (in the video attached, the 1st match, the 2nd and the 3rd doesn't, ...)


https://user-images.githubusercontent.com/5413702/114828524-4a462e00-9dca-11eb-9915-d4f9c28f5683.mov

Comment (hkang1) [reply webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpHeatMaps.tsx]: oops, missed that interaction. Thanks for catching it!

**Review by apizzini — APPROVED** [review]
lgtm!

**Pull request merged**